package.json

```diff
@@ -17,17 +17,24 @@
   "author": "WalletConnect, Inc. <walletconnect.com>",
   "homepage": "https://github.com/walletconnect/walletconnect-monorepo/",
   "license": "Apache-2.0",
+  "workspaces": [
+    "packages/types",
+    "packages/utils",
+    "packages/core",
+    "packages/sign-client",
+    "providers/signer-connection",
+    "providers/ethereum-provider"
+  ],
   "scripts": {
-    "clean": "lerna clean --yes",
-    "bootstrap": "lerna bootstrap --hoist",
-    "lint": "lerna run lint",
-    "build": "lerna run build",
-    "test": "lerna run test",
-    "check": "run-s lint build test",
-    "reset": "run-s clean bootstrap check",
-    "commit-version": "node ops/js/commit-version.js",
+    "clean": "npm run clean --workspaces",
+    "lint": "npm run lint --workspaces",
+    "prettier": "npm run prettier --workspaces",
+    "build": "npm run clean --workspaces; npm run build --workspaces",
+    "test": "npm run test --workspaces",
+    "check": "npm run lint; npm run build; npm run test",
+    "reset": "npm run clean; npm run check",
     "new-version": "lerna version --no-git-tag-version",
-    "pre-publish": "run-s new-version reset commit-version",
+    "pre-publish": "npm run new-version; npm run reset",
```

Comment:
Why was the `commit-version` step removed here?

Reply:
I looked at this during my cleanup work and I don't think it ever worked i.e. I can't find these commits in git history for last few versions we pushed. Plus when we release we create version tags manually as well as push a separate pr with updated version. Next steps with this would be to actually not run releases manually, rather trigger github action when we "tag" a release here. I'll be looking into this after this pr is merged.

Reply:
> I can't find these commits in git history for last few versions we pushed.

Not sure what you mean, I think it was just a convenience script to commit the version after lerna updates everything like here (but I manually tagged it afterwards): https://github.com/WalletConnect/walletconnect-monorepo/commit/419ac1815ac93482b6a19274328af3c0416d5746

I'm not attached to it and agree we should replace with something that also does the tag for us automatically, just wasn't clear on why you thought it wasn't working 🤔

Reply:
@bkrem yeh so basically that didn't happen to any of the releases I made i.e. lerna did it's thing, but I had to manually commit / pr them in still. And afterwards chad usually manually tagged them. So I got rid of it. My plan on what to do next is basically this

1. We tag version on `v2` branch
2. This triggers ci run that runs lerna with pre-defined version (i.e. tag)
3. This is then deployed to npm
4. Comited back to v2 branch